Mathlib/AlgebraicGeometry/Morphisms/QuasiCompact.lean

```diff
@@ -181,7 +181,7 @@ theorem QuasiCompact.affineProperty_isLocal : (QuasiCompact.affineProperty : _).
       topIsAffineOpen _]
 #align algebraic_geometry.quasi_compact.affine_property_is_local AlgebraicGeometry.QuasiCompact.affineProperty_isLocal
 
-theorem QuasiCompact.affine_openCover_tFAE {X Y : Scheme.{u}} (f : X ⟶ Y) :
+theorem QuasiCompact.affine_openCover_tfae {X Y : Scheme.{u}} (f : X ⟶ Y) :
```

Comment:
I see that we use both `tfae` and `TFAE`... do you know if there is an "official" decision about this?

Reply:
Zulip seems to lean towards `tfae` at the moment, so I'll leave it for now